website/docs/d/iam_policy_assignment.html.markdown

```diff
@@ -0,0 +1,83 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_iam_policy_assignment"
+description: |-
+  Get information about policy_assignments
+subcategory: "IAM Policy Management"
+---
+
+# ibm_iam_policy_assignment
+
+Provides a read-only data source to retrieve information about policy_assignments. You can then reference the fields of the data source in other resources within the same configuration by using interpolation syntax.
+
+## Example Usage
+
+```hcl
+data "ibm_iam_policy_assignments" "policy_assignment" {
+	account_id = "account_id"
+}
+```
+
+## Argument Reference
+
+You can specify the following arguments for this data source.
+
+* `accept_language` - (Optional, String) Language code for translations* `default` - English* `de` -  German (Standard)* `en` - English* `es` - Spanish (Spain)* `fr` - French (Standard)* `it` - Italian (Standard)* `ja` - Japanese* `ko` - Korean* `pt-br` - Portuguese (Brazil)* `zh-cn` - Chinese (Simplified, PRC)* `zh-tw` - (Chinese, Taiwan).
```

Comment:
we don't have this defined in schema  of "ibm_iam_policy_assignments" datasource